database/migrations/2017_08_06_153909_add_user_type_to_users.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddUserTypeToUsers extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('users', function (Blueprint $table) {
+            $table->string('user_type',50)->default('family');
+        });
+    }
```

Comment:
This file needs a `down()` method that drops this column in the `users` table in order for the tests to run. There is no rollback option without it